aicsimageio/dimensions.py

```diff
@@ -25,11 +26,25 @@ class DimensionNames:
 DEFAULT_DIMENSION_ORDER_LIST_WITH_SAMPLES = DEFAULT_DIMENSION_ORDER_LIST + [
     DimensionNames.Samples
 ]
+DEFAULT_DIMENSION_ORDER_LIST_WITH_MOSAIC_TILES = [
+    DimensionNames.MosaicTile
```

Comment:
I'm sure this does what you intend but this whole construct of adding arrays of stuff together is a little non-obvious. It could be a python thing I'm not used to but the behavior seems highly prone to what the contents of the variables are.

Reply:
Both you and @toloudis both commented on these. Does this seem better:

```python
DEFAULT_DIMENSION_ORDER_LIST_WITH_MOSAIC_TILES = [
    DimensionNames.MosaicTile,
    *DEFAULT_DIMENSION_ORDER_LIST
]
```

Basicaly doing list expansion rather than list addition.

Reply:
I don't mind the concatenation, either one is fine.  My thing is just that these names get long especially when you need to combine several of them! This is not a "must change" thing from my end.